README.md

```diff
@@ -56,7 +56,7 @@ The entire spec is contained in the [schema.json](src/schema.json) file.
 
   ```
   methods: {
-    <method name : [ input(s), output(s), minimum required outputs, 'latest' tag default position (if any) ] >,
+    <method name : [ Array Input(s), Array Output(s), String Output type, Integer Minimum required parameters, Boolean Requires default block parameter] >,
```

Comment:
I misspoke - the second element should be `Array Output(s)`, and so I the `String Output Type` is extraneous (e.g. `eth_getFilterLogs` returns `["FilterChanges"]`.